.gitignore

```diff
@@ -1,2 +1,3 @@
 .tmp/
 node_modules/
+/dist_dev/
```

Comment:
nit: alphabetize & remove the leading slash
```
.tmp/
dist_dev/
node_modules/
```

Reply:
> remove the leading slash

Sure, I don't care too much. But I am curious - why remove the leading slash? Seems like it would make sense to be more specific unless we expect to have multiple

Reply:
Consistency with the existing entries, more than anything.